optuna/integration/chainermn.py

```diff
@@ -189,6 +189,17 @@ def __init__(self, trial, comm):
         self.delegate = trial
         self.comm = comm
 
+    def suggest_float(self, name, low, high, *, log=False):
+        # type: (str, float, float, bool) -> float
+
+        def func():
+            # type: () -> float
+
+            assert self.delegate is not None
+            return self.delegate.suggest_float(name, low, high)
```

Comment:
It seems like the keyword argument of `log` is ignored.

Isn't it better to pass `log` to `self.delegate.suggest_float`?